ONSdigital/eq-author
# 648 front end changes for duplicating sections

### What is the context of this PR?
Front end changes made for the functionality of duplicating a section. Added duplicate button on section page and functionality behind it

### How to review 
1. Run tests to make sure they pass
1. Make sure you can duplicate a section with questions, answers and validations.

### Dependencies
- [x] - https://github.com/ONSdigital/eq-author-api/pull/150
